ui/app/src/App.tsx

```diff
@@ -36,7 +37,7 @@ function App(): ReactElement {
       }}
     >
       {location.pathname !== SignInRoute && location.pathname !== SignUpRoute && <Header />}
-
+      <BannerInfo></BannerInfo>
```

Comment:
But put it after `<Header />` component, we don't want to display this message on login or signup pages 
```suggestion
      <BannerInfo/>
```